src/helm/config/model_deployments.yaml

```diff
@@ -724,6 +724,18 @@ model_deployments:
 
   ## GPT 4 Models
 
+  - name: openai/gpt-4-1106-preview
+    model_name: openai/gpt-4-1106-preview
+    tokenizer_name: openai/cl100k_base
+    max_sequence_length: 128000
```

Comment:
From https://help.openai.com/en/articles/8555510-gpt-4-turbo, "introduces a 128k context window", but "the maximum number of output tokens for this model is 4096."

Reply:
Add a TODO comment for this, and link to #2098?

I don't think this affects anything in practice because the number of generated tokens is usually much shorter.